### PR TITLE
'subtask1'

### PR DESCRIPTION
**Pull request by pako535**

### CODE REVIEW

**Comment by tdiak** [comment]
Oh... You need to add postgres-data to .gitignore. Now You need to remove it from tracking:
https://stackoverflow.com/questions/7927230/remove-directory-from-remote-repository-after-adding-them-to-gitignore?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa

File: config.py [hunk 1]
```diff
@@ -10,6 +10,7 @@ class Config(object):
 class DevConfig(Config):
     DEBUG = True
     DATABASE_URI = getenv('DEV_DATABASE_URI', '')
```
Comment (tdiak) [inline config.py]: We are using:
SQLALCHEMY_DATABASE_URI

so we can remove DATABASE_URI property

File: tests/test_response.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from urllib import request
+
+
+class TestClass(object):
+
+    def test_response_200(self):
```
Comment (etude-ist) [inline tests/test_response.py]: Let's rewrite using proper pytest tools.

File: tests/test_response.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        try:
+            response = request.urlopen('http://localhost:5000/')
+            response = response.getcode()
+            if response == 200:
+                assert True
+            else:
+                assert False
+        except:
```
Comment (etude-ist) [inline tests/test_response.py]: It is a good practice to be specific about exception.

File: tests/test_response.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+from urllib import request
+
+
+class TestClass(object):
+
+    def test_response_200(self):
+        try:
+            response = request.urlopen('http://localhost:5000/')
+            response = response.getcode()
+            if response == 200:
```
Comment (etude-ist) [inline tests/test_response.py]: This if ... else ... could be easily eliminated.

Comment (pako535) [reply tests/test_response.py]: I know but it was supposed to be a simple test

**Pull request closed without merging**